### PR TITLE
Mvdb/remove print statements

### LINKED ISSUE #7: Can we remove the print statements all over the SDK ?
For instance [here](https://github.com/xplenty/xplenty.py/blob/master/xplenty/xplenty_api.py#L246-L301)

And maybe leverage python logging system :)

### LINKED ISSUE #8: Proper error handling
Hi guys,

Can we start having error handling in the SDK ? For instance [here](https://github.com/xplenty/xplenty.py/blob/master/xplenty/xplenty_api.py#L258-L260) the code shouldn't fail silently.

### PR DESCRIPTION
**Pull request by m-vdb**
Fixes #7 
Fixes #8

### CODE REVIEW

**Comment by m-vdb** [comment]
@motymichaely @saggineumann Hey guys! I wrote a little but of code to fix 2 issues I opened earlier on this repo. Do you think you could take a look? Also do you think you could do a pypi release right after? (cf. #9)

File: xplenty/exceptions.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import json
+
+
+class XplentyAPIException(Exception):
+    """
+    An exceprtion that is raised whenever an error occured.
```
Comment (olivierlefloch) [inline xplenty/exceptions.py]: exception
occurred

**Comment by olivierlefloch** [comment]
Slightly more legible without the whitespace: https://github.com/xplenty/xplenty.py/pull/10/files?w=b

**Comment by motymichaely** [comment]
@m-vdb  @olivierlefloch - Looks good!

**Comment by motymichaely** [comment]
@m-vdb - With regards to Pypi - We want to add tests to this repo and make sure it's ready to be published as a package. Any contribution would be great!

**Pull request merged**